### DISCUSSION TITLE
Rotor_inside_visualize_cut_plane

### DISCUSSION THREAD

**Discussion by DanieleParis** [NatLabRockies/floris#523] [Q&A] [answered]
Hi, I'd like to know how can i put inside the image of the visuliaze_cut_planethe rotor of the turbine, in such a way to have a clear understanding of the yaw angle of the turbine. 
Below there is my code and the image that I have if I run the script.

```python
import matplotlib.pyplot as plt
from floris.tools import FlorisInterface
import numpy as np
from floris.tools.visualization import visualize_cut_plane


fi = FlorisInterface("inputs/gch.yaml")
X, Y =[(0.0, 800), (0.0, 0.0)]

yaw_angles = np.zeros([1, 1, 2])


yaw_angles[:,:,0] = 25

fig, ax = plt.subplots()
fi.reinitialize(layout=(X, Y))
horizontal = fi.calculate_horizontal_plane(x_resolution=500, yaw_angles=yaw_angles, y_resolution=500, height=90)
visualize_cut_plane(horizontal, ax=ax, cmap="coolwarm", color_bar=True)
ax.set_xlabel('x'); ax.set_ylabel('y')
plt.show()
```

![immagineGithub](https://user-images.githubusercontent.com/117908449/201073424-d3e05d9b-9f42-403f-b6f4-5acfef57986e.png)

While I'd like to have an image like this, with the rotor in black.

![magaricosi](https://user-images.githubusercontent.com/117908449/201074709-ee1d2df9-945b-40a4-af36-bae45c2a5053.png)

**Comment by rafmudaf** [comment, marked as answer]
Hi @DanieleParis I recently updated #512 to fix an issue with the rotor plotting and include it in an example. Please see the changes in [the example](https://github.com/NREL/floris/pull/512/files#diff-beca49efc186bce3a1003fbcb7ec776394b86b0779372eb591a2f0e6cacb6f9fR64) for how to use the function. Since it is not yet merged into the main repository, you might have to wait until next week or pull the branch from my repository (branch is `api` and my fork is https://github.com/rafmudaf/floris).

I hope this helps!
